### ISSUE TITLE
Add W&B example

### ISSUE THREAD

**Issue by jafermarq** [adap/flower#1026]
### Describe the type of feature and its functionality.

In a few of the projects I've been involved I found it quite useful to integrate [Weight&Biases](https://wandb.ai/site) in the FL training process. Would there be interest in adding an example using W&B to the `examples/` directory?

### Describe step by step what files and adjustments are you planning to include.

The idea would be to build upon the `simulation_pytorch` example and modify it (or create a new example) by adding W&B at the server-level and client-level. The elements that this example will include are:

-    Basics about W&B biases (i.e. creating an account etc)
-    W&B integration in the server so metrics such as global validation accuracy/loss at the end of each epoch are logged.
-    Similarly, the server could log client-level specific metrics that has received in a given round.
-    Finally, the example would show how all the data logged can be conveniently analysed in a W&B dashboard.
-    I'd include demo code and link to a public W&B dashboard (so people can check what to expect from running the example).
-    (potentially) I could add a more advance section in the example showing how to use W&B's [sweep](https://docs.wandb.ai/guides/sweeps/quickstart) functionality.

**Comment by prathapkumarbaratam** [comment]
@jafermarq  hi , have you done implementation of basic pytorch example? if so .can you provide link for that?

**Comment by jafermarq** [comment]
here draft PR: https://github.com/adap/flower/pull/2239
